database/abusescanner.go

```diff
@@ -31,6 +32,15 @@ const (
 
 	// lockTTL is the time-to-live in seconds for a lock
 	lockTTL = 300 // 5 minutes
+
+	// mongoTestUsername is the username used to connect with the test DB.
+	mongoTestUsername = "admin"
+
+	// mongoTestPassword is the password used to connect with the test DB.
+	mongoTestPassword = "aO4tV5tC1oU3oQ7u"
+
+	// mongoTestConnString is the connection string to connect with the test DB.
+	mongoTestConnString = "mongodb://localhost:37017"
```

Comment:
I thought about that when writing it but you get an annoying cyclic reference. Because you still want to return the `DB` struct from the database package, and you want to use your test db from your database package. So that yields a cyclic ref. which is annoying.

I don't see the big deal with `database.NewTestDatabase()` anyway, I think it's nice and it works.

Reply:
Aw, that's a bummer. Sure, let's keep it here then. Not ideal but cycling deps are a PITA.